### ISSUE TITLE
NVDA does'nt announce the level of the node in java tree views

### ISSUE THREAD

**Issue by mohammad-suliman** [nvaccess/nvda#5766]
NVDA doesn't announce the level of tree nodes within java swing apps.
steps to reproduce:
* download swingset2:
https://java.net/projects/ojt-accessibility/downloads/directory/Demos
* run the downloaded jar file and NVDA as well
* from the toolbar, click the JTree demo toggle button
* navagate to the tree view and move the focus between the nodes using arrow keys

expected result:
the node content is announced besides the level also
observed result:
the level is not announced

NVDA version: 2015.4
windows 7 64bit
JDK 8u74

note:
there is another issue here with expand / collaps events of the node. think reporting this in other separate issue is better in a case this problem is NVDA specific

**Comment by jcsteh** [comment]
[This post on nvda-devel provides some useful technical info about how to fix this](http://nabble.nvda-project.org/tree-views-within-java-applications-tp39711p39736.html). It'd make sense to look at the expanded/collapsed state change here as well, but we might file another issue for that if it's nontrivial.

**Comment by nvaccessAuto** [comment]
Incubated in 4ec1fd1012449c92f01b9a78fd50f865311db4c6.

**Comment by rpaquay** [comment]
Note: I opened bug #5812 to address the issue of expand/collapse events not announced.
